apps/betterangels-backend/shelters/enums.py

```diff
@@ -64,7 +64,11 @@ class GeneralServiceChoices(models.TextChoices):
 
 
 class HealthServiceChoices(models.TextChoices):
+    DENTAL = "dental", _("Dental")
+    GENERAL = "general", _("General")
+    HEALTH_LINKAGES = "health_linkages", _("Health Linkages")
     MEDICATION_ADMINISTRATION = "medication_administration", _("Medication Administration")
+    MEDICATION_MANAGEMENT = "medication_management", _("Medication Management")
```

Comment:
not a code question, just curious: what's the difference between medication management and medication monitoring?

Reply:
Not exactly sure, but it was requested by Bowie